draft-thomson-httpbis-http2bis.xml

```diff
@@ -1167,231 +1162,77 @@
-            limited.
+            A thorough description of the RFC 7540 priority scheme remains in <xref target="RFC7540"
+            section="5.3"/>.
           </t>
           <t>
-            The amount of additional state an endpoint maintains for prioritization could be
-            dependent on load; under high load, prioritization state can be discarded to limit
-            resource commitments.  In extreme cases, an endpoint could even discard prioritization
-            state for active or reserved streams. If a limit is applied, endpoints SHOULD maintain
-            state for at least as many streams as allowed by their setting for
-            <xref target="SETTINGS_MAX_CONCURRENT_STREAMS" format="none">SETTINGS_MAX_CONCURRENT_STREAMS</xref>.  Implementations SHOULD also attempt to
-            retain state for streams that are in active use in the priority tree.
+            Signaling priority information could still be necessary to attain good performance.
+            Where signaling priority information is important, endpoints are encouraged to use an
+            alternative scheme, such as <xref target="I-D.ietf-httpbis-priority"/>.
           </t>
           <t>
-            If it has retained enough state to do so, an endpoint receiving a
-            <xref target="PRIORITY" format="none">PRIORITY</xref> frame that changes the priority of a
-            closed stream SHOULD alter the dependencies of the streams that depend on it.
+            Though the priority signaling from RFC 7540 was not widely adopted, the information it
+            provides can still be useful in the absence of better information.  Endpoints that
+            receive <xref target="HEADERS" format="none">HEADERS</xref> or <xref target="PRIORITY"
+            format="none">PRIORITY</xref> frames can benefit from applying any priority that is
+            present.  In particular, implementations that consume these signals would not benefit
+            from discarding these priority signals in the absence of alternatives.
           </t>
-        </section>
-        <section anchor="pri-default">
-          <name>Default Priorities</name>
           <t>
-            All streams are initially assigned a non-exclusive dependency on stream 0x0.  <xref target="PushResources">Pushed streams</xref> initially depend on their associated
-            stream.  In both cases, streams are assigned a default weight of 16.
+            Servers SHOULD use other contextual information in determining priority of requests in
+            the absence of any priority signals.  In particular, servers might need to decide
+            whether to interpret the complete absence of signals as an indication that the client
+            has not implemented the feature or that the client that has deliberately chosen to rely
+            on the defaults described in <xref target="RFC7540" section="5.3.5"/>.
```

Comment:
I wonder if this is a good thing to say, as it has been shown that the defaults in RFC7540 (fair round robin) are typically the worst-case scenario for web page loading performance. 

Does it not make more sense to deprecate these defaults as well and instead recommend a FIFO/sequential scheme instead in the absence of any signals (or just refer to the extensible priorities draft for the better default options (which also uses FIFO))?

Reply:
Good call.  I was being conservative, but I think that we could easily make a stronger statement here.